clang/runtime/dpct-rt/include/dpct/bindless_images.hpp

```diff
@@ -314,10 +314,10 @@ template <class T, int dimensions> class bindless_image_wrapper {
 
   /// Attach linear data to bindless image.
   /// \param [in] data The linear data used to create bindless image.
-  /// \param [in] channel The image channel used to create bindless image.
   /// \param [in] size The size of linear data used to create bindless image.
+  /// \param [in] channel The image channel used to create bindless image.
   /// \param [in] q The queue where the image creation be executed.
-  void attach(void *data, const image_channel &channel, size_t size = UINT_MAX,
+  void attach(void *data, size_t size, const image_channel &channel,
```

Comment:
what is the reason for the reorder?

Reply:
Because the old implementation in image.hpp has the same functionality, we need keep them consistent to reuse migration logic.